Source/AwsCommonRuntimeKit/crt/CommonRuntimeError.swift

```diff
@@ -1,6 +1,29 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0.
 
+import AwsCCommon
+import Foundation
+
 public enum CommonRunTimeError: Error {
     case crtError(CRTError)
 }
+
+public struct CRTError: Equatable {
+    public let code: Int32
+    public let message: String
+    public let name: String
+
+    public init<T: BinaryInteger>(code: T) {
+        if code > INT32_MAX || code <= 0 {
+            self.code = AWS_OP_ERR // Error Unknown
```

Comment:
trivial: ~AWS_OP_ERR~ -> `AWS_ERROR_UNKNOWN` or set `-1` if you really want to set something different

AWS_OP_ERR and AWS_OP_SUCCESS are not themselves error codes. They're the values we return from a function that has set a thread-local error-code

Reply:
Thanks, resolved.